data/metrics/src/testFixtures/java/tech/pegasys/teku/metrics/StubMetricsSystem.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.metrics;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.DoubleSupplier;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.Counter;
+import org.hyperledger.besu.plugin.services.metrics.LabelledMetric;
+import org.hyperledger.besu.plugin.services.metrics.MetricCategory;
+import org.hyperledger.besu.plugin.services.metrics.OperationTimer;
+
+public class StubMetricsSystem implements MetricsSystem {
+
+  private Map<MetricCategory, Map<String, StubCounter>> counters = new ConcurrentHashMap<>();
+  private Map<MetricCategory, Map<String, StubGauge>> guages = new ConcurrentHashMap<>();
```

Comment:
nit:
```suggestion
  private Map<MetricCategory, Map<String, StubGauge>> gauges = new ConcurrentHashMap<>();
```

Reply:
ugh, I hate that word! Get the spelling wrong all the time....